Feature/kin 114 back fastapi api gateway

Student-Jasons opened the pull request:
1. CORS 설정하여 /proxy경로는 로컬호스트로만 접속 가능하게 변경
2. celery 디버깅시 기존 로거 사용으로 변경

hwanju1596 committed:
✨ proxy 추가

Student-Jasons committed:
🎨 celery 서버 모드에서만 커스텀 로거 사용

Student-Jasons committed:
Merge remote-tracking branch 'origin/feature/KIN-114-Back-Fastapi-api-gateway' into feature/KIN-114-Back-Fastapi-api-gateway

Student-Jasons committed:
:bug: 호출시 무한재귀에 빠지는 문제 해결

restyled-commits committed:
Restyled by autopep8

restyled-commits committed:
Restyled by black

restyled-commits committed:
Restyled by isort

restyled-commits committed:
Restyled by pyment

restyled-commits committed:
Restyled by reorder-python-imports

restyled-commits committed:
Restyled by whitespace

restyled-commits committed:
Restyled by yapf

Student-Jasons committed:
Restyle Feature/kin 114 back fastapi api gateway (#53)

Automated style fixes for #52, created by [Restyled][].

The following restylers [made
fixes](https://restyled.io/gh/king-wang-jjang/repos/kingwangjjang-be/jobs/4722440):

- [autopep8](https://github.com/hhatto/autopep8)
- [black](https://github.com/python/black)
- [isort](https://github.com/timothycrosley/isort/)
- [pyment](https://github.com/dadadel/pyment)
-
[reorder-python-imports](https://github.com/asottile/reorder_python_imports)
-
[whitespace](https://github.com/restyled-io/restyler-whitespace/blob/master/README.md)
- [yapf](https://github.com/google/yapf)


To incorporate these changes, merge this Pull Request into the original.
We
recommend using the Squash or Rebase strategies.


**NOTE**: As work continues on the original Pull Request, this process
will
re-run and update (force-push) this Pull Request with updated style
fixes as
necessary. If the style is fixed manually at any point (i.e. this
process finds
no fixes to make), this Pull Request will be closed automatically.

Sorry if this was unexpected. To disable it, see our [documentation][].

[restyled]: https://restyled.io
[documentation]:
https://github.com/restyled-io/restyled.io/wiki/Disabling-Restyled

restyled-commits committed:
Restyled by black

restyled-commits committed:
Restyled by isort

restyled-commits committed:
Restyled by pyment

restyled-commits committed:
Restyled by reorder-python-imports

restyled-commits committed:
Restyled by whitespace

restyled-commits committed:
Restyled by yapf

hwanju1596 commented on app/routes/path/ApiPaths.py:
```diff
@@ -1,9 +1,11 @@
 ## Graphql
 GRAPHQL = "/graphql"
-
+GRAPHQL_WITH_PROXY = PROXY + GRAPHQL
```
with proxy는 없어도 될 것 같아요

hwanju1596 replied:
ROOT = "/proxy"

GRAPHQL = ROOT + SEPATATE + "graphql

hwanju1596 replied:
이게 낫지 않을까요?

Student-Jasons replied:
네 알겠습니다

Student-Jasons committed:
Restyle Feature/kin 114 back fastapi api gateway (#54)

Automated style fixes for #52, created by [Restyled][].

The following restylers [made
fixes](https://restyled.io/gh/king-wang-jjang/repos/kingwangjjang-be/jobs/4722452):

- [black](https://github.com/python/black)
- [isort](https://github.com/timothycrosley/isort/)
- [pyment](https://github.com/dadadel/pyment)
-
[reorder-python-imports](https://github.com/asottile/reorder_python_imports)
-
[whitespace](https://github.com/restyled-io/restyler-whitespace/blob/master/README.md)
- [yapf](https://github.com/google/yapf)


To incorporate these changes, merge this Pull Request into the original.
We
recommend using the Squash or Rebase strategies.


**NOTE**: As work continues on the original Pull Request, this process
will
re-run and update (force-push) this Pull Request with updated style
fixes as
necessary. If the style is fixed manually at any point (i.e. this
process finds
no fixes to make), this Pull Request will be closed automatically.

Sorry if this was unexpected. To disable it, see our [documentation][].

[restyled]: https://restyled.io
[documentation]:
https://github.com/restyled-io/restyled.io/wiki/Disabling-Restyled

Student-Jasons committed:
:art: apipath 수정

Student-Jasons committed:
:bug: poetry 관련 애러 수정

Student-Jasons committed:
:bug: poetry 관련 애러 수정

Student-Jasons committed:
:bug: poetry 관련 애러 수정
